### PR TITLE
docs: update react readme

### PR DESCRIPTION
**Pull request by beeme1mr**

### CODE REVIEW

File: packages/react/README.md [hunk 1]
```diff
@@ -15,26 +15,66 @@
...
+[OpenFeature](https://openfeature.dev) is an open specification that provides a vendor-agnostic, community-driven API for feature flagging that works with your favorite feature flag management tool or in-house solution.
+
+<!-- x-hide-in-docs-end -->
 
 🧪 This SDK is experimental.
 
-## Basic Usage
+## Overview
+
+The OpenFeature React SDK is a package that adds React-specific functionality to the [OpenFeature Web SDK](https://openfeature.dev/docs/reference/technologies/client/web).
```
Comment (toddbaert) [inline packages/react/README.md:41]:
```suggestion
The OpenFeature React SDK adds React-specific functionality to the [OpenFeature Web SDK](https://openfeature.dev/docs/reference/technologies/client/web).
```

File: packages/react/README.md [hunk 2]
```diff
@@ -15,26 +15,66 @@
...
+
+In addition to the feature provided by the [web sdk](https://openfeature.dev/docs/reference/technologies/client/web), additional capabilities include:
```
Comment (toddbaert) [inline packages/react/README.md:43]:
```suggestion
In addition to the feature provided by the [web sdk](https://openfeature.dev/docs/reference/technologies/client/web), capabilities include:
```

Redundant I think

**Review by toddbaert — APPROVED** [review]
Approved with minor textual recommendations

**Review by lukas-reining — APPROVED** [review]
Looks good!

**Pull request merged**